controls/cis_ocp_1_3_0/section-1.yml

```diff
@@ -329,8 +329,12 @@ controls:
     controls:
     - id: 1.3.1
       title: Ensure that garbage collection is configured as appropriate
-      status: partial
-      rules: [] 
+      status: Completed
```

Comment:
```
  RuntimeError: Unable to parse controls from /home/runner/work/content/content/controls/cis_ocp_1_3_0.yml: The given status 'Completed' in the control '1.3.1' was invalid. Please use one of the following: ['automated', 'documentation', 'does not meet', 'inherently met', 'manual', 'not applicable', 'partial', 'pending', 'planned', 'supported']
```
`Completed` is not supported. Please use one of the supported options.

Reply:
@rutvik23 is this one of the controls that are going away? In that case I would set the status as pending for now and add a note that this one is going away

Reply:
@jhrozek yes, (if i recall correctly) few rules in this control are going away but the rules with  _master _worker are going to stay. As per new CIS revision, we are also moving this rule into different control section altogether, hence keeping this in `Pending` state to be re-considered for future developments.